awx/settings/defaults.py

```diff
@@ -71,6 +71,7 @@ def IS_TESTING(argv=None):
 # timezone as the operating system.
 # If running in a Windows environment this must be set to the same as your
 # system time zone.
+USE_TZ = False
```

Comment:
Will this change timezone naivety or something for DRF date fields?  What changed here in Django/DRF to necessitate this setting change?